api/src/test/java/org/openmrs/module/attachments/AttachmentsServiceTest.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.attachments;
+
+import org.junit.Test;
+import org.openmrs.Encounter;
+import org.openmrs.Patient;
+import org.openmrs.Visit;
+
+public class AttachmentsServiceTest {
+	
```

Comment:
Have you tried?
```java
@Autowired
private AttachmentsService as;
```
Doing so will highlight the Spring bean configuration is missing.